flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/AbstractHDFSWriter.java

```diff
@@ -65,7 +65,7 @@ public void configure(Context context) {
       try {
         timeBetweenCloseRetries = context.getLong("hdfs.callTimeout", 10000l);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lower case "l" long suffix by an upper case "L". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ALowerCaseLongSuffixCheck)